sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/StorageSpec.groovy

```diff
@@ -48,7 +48,7 @@ class StorageSpec extends Specification {
         interceptorManager.close()
     }
 
-    protected static TestEnvironment getEnv() {
+    protected static TestEnvironment getEnvironment() {
```

Comment:
I had to rename it as it started to conflict with built in `env`/`getEnv` that was yielding just hashmap of env variables...